.github/pull_request_template.md

```diff
@@ -1,14 +1,3 @@
 ## Problem
 
 ## Summary of changes
```

Comment:
I think we should get rid of these as well.

Their aim was to stop people pushing empty commit messages, now we have a lot of full template commit messages or people still leaving these headings in. Not even github supports markdown for commit messages. `git` certainly does not.